source/Editor/Triggers/Plugin_CustomHeightDisplayTrigger.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Xna.Framework;
+
+namespace Snowberry.Editor.Triggers;
+
+[Plugin("everest/customHeightDisplayTrigger")]
+public class Plugin_CustomHeightDisplayTrigger : Trigger {
+    [Option("vanilla")] public bool Vanilla = false;
+    [Option("target")] public float Target = 0;
+    [Option("from")] public float From = 0;
+    [Option("text")] public string text = "{x}m";
+    [Option("progressAudio")] public bool ProgressAudio = false;
+    [Option("displayOnTransition")] public bool DisplayOnTransition = false;
+
+    public override void Render() {
+        base.Render();
+
+        var postfix = (text.IndexOf("}") != text.Length) ? text.Substring(text.LastIndexOf("}") + 1) : "m";
```

Comment:
should check for `{x}`, not just `}` to avoid letting invalid triggers appear to work

Reply:
i'm just gonna redo this section to allow for prefix support (ie if you wanted to do Entering {500}m)